includes/managers/widgets.php

```diff
@@ -240,9 +256,16 @@ public function register( Widget_Base $widget_instance ) {
 	 * @return void
 	 */
 	private function register_promoted_widgets() {
-		if ( Plugin::$instance->experiments->is_feature_active( 'nested-elements' ) ) {
-			$nested_tabs = new NestedTabs();
-			$this->_widget_types = [ $nested_tabs->get_name() => $nested_tabs ] + $this->_widget_types;
+
+		foreach ( $this->_promoted_widgets as $module_name => $class_name ) {
+
+			if ( Plugin::$instance->experiments->is_feature_active( $module_name ) ) {
+				$instance = new $class_name();
+
+				$this->_widget_types = [
+					$instance->get_name() => $instance,
+				] + $this->_widget_types;
+			}
```

Comment:
```
private function register_promoted_widgets() {
	foreach ( $this->_promoted_widgets as $module_name => $class_name ) {
		if ( Plugin::$instance->experiments->is_feature_active( $module_name ) ) {
			$instance = new $class_name();
			$this->_widget_types[ $instance->get_name() ] = $instance;
		}
	}

	$this->_widget_types = array_merge( $this->_widget_types, $this->_widget_types );
}
```
@davseve WDYT?

Reply:
Maybe `$this->_widget_types` needs to be instantiated as an empty array. So `[]` instead of `null`